modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/DefaultExecutionMediator.java

```diff
@@ -101,6 +103,7 @@ public void execute(CompletableComponentExecutor<M> executor,
 
     final MutableConfigurationStats stats = getMutableConfigurationStats(context);
     if (stats != null) {
+      stats.addActiveComponent();
       stats.addInflightOperation();
```

Comment:
Shouldn't this be removed? You already counted the active component. Why count twice?

Reply:
They count different things, I left the old one to preserve the ConfigurationStats api which states:
  /**
   * @return How many currently executing operations are making use of the referenced configuration
   *
   * @deprecated Use {@link #getActiveComponents()} instead.
   */
  @Deprecated
  int getInflightOperations();